docs/book/src/cronjob-tutorial/testdata/project/main.go

```diff
@@ -34,10 +34,10 @@ import (
 
 /*
 The first difference to notice is that kubebuilder has added the new API
-group's package (`kbatchv1beta1`) to our scheme.  This means that we can use those
+group's package (`batchv1`) to our scheme.  This means that we can use those
 objects in our controller.
 
-We'll also need to add the kubernetes batch v1 scheme, since we're creating
+We'll also need to add the kubernetes batch v1 (`kbatchv1beta1`) scheme, since we're creating
```

Comment:
It should be `kbatchv1` here. You please also update the import path to `kbatchv1` in line 23.

Reply:
Good catch, done!